Java/Rahul/Day13QueeStack.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Stack;
+
+public class MyQueeStack {
+
+    public Stack<Integer> queeStack;
+
+    public int size;
+
+    public int lastElement;
+
+    public MyQueeStack(){
+        queeStack = new Stack<>();
+        size =0;
+    }
+
+    public int size(){
+        return size;
+    }
+
+    public boolean isEmpty(){
+        return size == 0;
+    }
+
+    public void add(Integer value){
+        if(size == 0)
+            lastElement = value;
+
+        queeStack.add(value);
+        size++;
+    }
+
+    public Integer element() throws Exception {
+        if(size == 0)
+            throw new Exception("empty quee");
+
+        return lastElement;
+    }
+
+    public Integer peek()  {
+        if(size == 0)
+            return null;
+
+        return lastElement;
+    }
+
+    public Integer poll()  {
+        if(size == 0)
+            return null;
+
+        Stack<Integer> temp = new Stack<>();
+
+        while(!queeStack.isEmpty()){
+            temp.add(queeStack.pop());
+        }
+        int op = temp.pop();
+        lastElement =temp.peek();
+        while(!temp.isEmpty()){
```

Comment:
just a suggestion --- may be we can avoid this re-transfer to main stack by using two stacks instead of 1 main stack and 1 temp stack.

Reply:
how Rathu